tests/unit/compiler/test_opcodes.py

```diff
@@ -64,3 +65,14 @@ def test_get_opcodes(evm_version):
     else:
         for op in ("TLOAD", "TSTORE", "MCOPY"):
             assert op not in ops
+
+
+def test_build_opcodes():
+    assert _build_opcodes(bytes.fromhex("610100")) == "PUSH2 0x0100"
```

Comment:
i think it might be useful here to include an ambiguous example. for instance, i noticed that `592` and `9472` both get encoded with `PUSH2 0x250`